File: app/src/main/java/com/eba/appgastos/ui/gastosFijos/GastosFijosViewModel.java
package com.eba.appgastos.ui.gastosFijos;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class GastosFijosViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public GastosFijosViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is notifications fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/ed/app_gastos/ui/gastosFijos/GastosFijosViewModel.kt
package com.ed.app_gastos.ui.gastosFijos

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class GastosFijosViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is notifications Fragment"
    }
    val text: LiveData<String> = _text
}